### ISSUE TITLE
Two problems using windows key to alt-tab from the game (may happen with alt-tab too, I have it bound elsewhere)

### ISSUE THREAD

**Issue by ntrna** [chromiecraft/chromiecraft#1663]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

(This is not in windowed mode.) When my character lands from a flight path into Ironforge while I am looking at the other program, I click back into wow from the taskbar and my character has fallen through the map. I have not noticed this at other flight paths.

Also, although most sound will continue in the background, the sound of my character striking the anvil usually does not. The sounds of smelting & of completing a craft, when it is added to my bags, do continue in the background.

### Expected Blizzlike Behaviour

Should be able to alt-tab while in flight, and not tab back into your character falling thru the map to their death.

Should be able to hear all sounds in the background while alt-tabbed.

### Steps to reproduce the problem

1. Fly an Alliance character to Ironforge. Alt-tab out while in flight. Tab back in after sufficient time has passed for character to have landed.
2. Go to an anvil. Start crafting something with blacksmithing. Alt-tab & listen for the sound of character striking anvil to continue or not.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/595bb6adccbabc714469f3935541978283b8bdfb

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by syssneck** [comment]
Interesting, we had another report of a player falling through the ground after dismounting from a flight to sw: https://github.com/azerothcore/azerothcore-wotlk/issues/7742
Thanks for the info! Tabbing in and out of the game might have something to do with it.

**Comment by iraizo** [comment]
> 
> 
> Interesting, we had another report of a player falling through the ground after dismounting from a flight to sw: [azerothcore/azerothcore-wotlk#7742](https://github.com/azerothcore/azerothcore-wotlk/issues/7742)
> Thanks for the info! Tabbing in and out of the game might have something to do with it.

I can confirm this, i alt tabbed and fell trough the ground after a flight from westfall to stormwind

**Comment by jamadaha** [comment]
Could reproduce the missing sound, but not the fall out of map.
So I'd wait for someone to reproduce it on AC.

**Comment by iraizo** [comment]
I tried to reproduce this again on CC but it only happens rarely or maybe theres a specific thing i did to make it happen.

**Comment by syssneck** [comment]
@jamadaha can you create an issue for the sound problem on AC then? I'm not sure if it is a client issue, but we can open an issue and see what others have to say. Thanks!!

We have already an issue for the falling through the map issue on AC (see my post above).

**Comment by ntrna** [comment]
Alt-tabbing also mutes the sound of other people blacksmithing nearby.

**Comment by UltraNix** [comment]
But do you have enabled "Sound in background"? It's normal if your Wow.exe is not an active window.

**Comment by Snufflenose** [comment]
I can confirm this bug. It happened twice to me already.

I took a gryphon ride to Ironforge (I think it was both times from Menethil) and alt-tabbed to desktop (or rather cmd-tab since I'm playing on Mac via Wine).

Once I tabbed back in after my char has landed, he died from fall damage.

After clicking "release spirit" my char got resurrected - the first time right in front of the Gates of Ironforge, the second time at the graveyard in eastern Dun Morogh.

It also said I can't log out afterwards.

![Bildschirmfoto 2021-10-09 um 15 08 40](https://user-images.githubusercontent.com/91610030/136659502-21dd2e7c-a522-412a-8ad0-6d667f63bba9.png)

![Bildschirmfoto 2021-10-09 um 15 21 38](https://user-images.githubusercontent.com/91610030/136659650-489b74ac-8e00-4130-adc8-ee85788ca46b.png)

![Bildschirmfoto 2021-10-09 um 15 35 32](https://user-images.githubusercontent.com/91610030/136660012-6e48698b-f128-45d7-a5cd-6312ec7d32b2.png)
